homeTask_10/TestUnitTestsForHome11/TestData.cs

```diff
@@ -0,0 +1,150 @@
+﻿using System;
+using System.Collections.Generic;
+using homeTask_10;
+
+
+namespace UnitTestsProject
+{
+    public class TestData
+    {
+        public List<Book> MixDataBookOrder()
+        {
+            List<Author> authors = new List<Author>();
+            authors.Add(new Author("Maksim", "Kuznets", new DateTime(1990, 7, 20)));
+            authors.Add(new Author("Eugen", "Martul", new DateTime(1991, 2, 1)));
+            List<Book> books = new List<Book>();
+            books.Add(new Book("Game Of Thrones", new DateTime(1901, 5, 6), 1443,
+                new List<Author>() { authors[0], authors[1], }
+                ));
+            books.Add(new Book("War and Peace", new DateTime(1888, 1, 16), 23,
+                new List<Author>() { authors[1], }
+                ));
+            books.Add(new Book("Collection of poems Bykov", new DateTime(1970, 2, 28), 9999,
+                new List<Author>() { authors[0] }
+                ));
+            books.Add(new Book("Ruslan & lyudmila", new DateTime(1980, 3, 28), 765,
+                new List<Author>() { authors[0] }
+                ));
+            books.Add(new Book("Idiot", new DateTime(1980, 3, 28), 765,
+                new List<Author>() { authors[1], }
+                ));
+            books.Add(new Book("Godfather", new DateTime(1977, 5, 28), 666,
+                new List<Author>() { authors[1], }
+                ));
+            return books;
+        }
+        public List<Book> RightDataBookOrder()
+        {
+            List<Author> authors = new List<Author>();
+            authors.Add(new Author("Maksim", "Kuznets", new DateTime(1990, 7, 20)));
+            authors.Add(new Author("Eugen", "Martul", new DateTime(1991, 2, 1)));
+            List<Book> rightOrderBook = new List<Book>();
+            rightOrderBook.Add(new Book("Collection of poems Bykov", new DateTime(1970, 2, 28), 9999,
```

Comment:
Will be better to create private array with values and use it instead duplicate code many times.